### PR TITLE
feat(smart actions): endpoint that handle forms' load hooks

### PR DESCRIPTION
**Pull request by rap2hpoutre**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

File: src/routes/actions.js [hunk 1]
```diff
@@ -92,31 +165,18 @@ class Actions {
...
-    // One route is created for each action which have a `hooks.load` property.
-    schema.actions.filter((action) => action.hooks && action.hooks.load)
-      .forEach((action) => {
-        app.post(
-          this.getRoute(action, 'hooks/load', options),
-          auth.ensureAuthenticated,
-          this.getHookLoadController(action, model, Implementation),
-        );
-      });
+    this.buildRoutes(schema.actions, options, auth);
```
Comment (GuillaumeCisco) [inline src/routes/actions.js:179]:
```suggestion
    this.buildRoutes(schema.actions);
```

File: src/routes/actions.js [hunk 2]
```diff
@@ -38,31 +38,67 @@ class Actions {
...
+   * @param {Object} action The smart action
+   * @returns {Function} A route callback for express
+   */
+  getHookLoadController(action) {
+    return async (request, response) => (
+      this.getHook(request, response, async (record) => this.smartActionHook.getResponse(
+        action.hooks.load,
+        action.fields,
+        record,
+      )));
```
Comment (GuillaumeCisco) [inline src/routes/actions.js:83]: I think I'm mistaking something.
The `getResponse` method signature is `getResponse(hook, record, fields)`.
Are `fields` and `record` inverted?

Same goes for change hook.

**Review by GuillaumeCisco — COMMENTED** [review]
Just a few comments.

Everything seems good!
Very nice work!
Clean and concise ✨

File: src/routes/actions.js (hunk 2 above)
Comment (rap2hpoutre) [reply src/routes/actions.js:83]: Wow! Nice catch, thanks!! The "fun" part is that all test passed since everything is mocked 😱 

Fixed!

**Review by GuillaumeCisco — APPROVED** [review]
LGTM 🐼

**Pull request merged**